src/Option/README.md

```diff
@@ -0,0 +1,24 @@
+# Option:
+
+Provides an Option component that is passed as a child to the ListBox component.
+
+## API
+
+### `aria-selected` (boolean; optional):
+The ListBox component will handle the `aria-selected` attribute itself, but the `aria-selected` prop can be used as an override, if necessary.
```

Comment:
I anticipated this being necessary based on similar patterns we have in [asInput](https://github.com/edx/paragon/blob/master/src/asInput/index.jsx#L81) and [CheckBox](https://github.com/edx/paragon/blob/master/src/CheckBox/index.jsx#L17). An example use case would be navigating away from the ListBox (say to a different tab or page) and needing to clear its selected state.